epoch_multinode_test.go

```diff
@@ -316,13 +394,19 @@ func (c *testComm) maybeTranslateOutoingToIncomingMessageTypes(msg *Message) {
 }
 
 func (c *testComm) Broadcast(msg *Message) {
+	c.lock.RLock()
```

Comment:
I think it makes sense to just make a function `isMessagePermitted` that RLocks around the message filter and not explicitly RLock everywhere we use it.